test/testdata/configs/gocritic.yml

```diff
@@ -2,9 +2,9 @@ linters-settings:
   gocritic:
     enabled-checks:
       - rangeValCopy
-      - flagDeref
-      - wrapperFunc
       - ruleguard
+    disabled-checks:
+      - dupSubExpr # So as not to overlap `ruleguard`.
```

Comment:
Couldnot understand this is regress or not 🤔

Reply:
I think it's related to the order of the rules (based on name)
I think it's better to change the test instead of the configuration.

Reply:
I refactored tests

- no intersection between default checkers and custom rules
- no intersection between gocritic-fixes tests (I was confused by `rangeValCopy` and `rangeExprVal`)
- little more border cases